Formula/aflplusplus.rb

```diff
@@ -0,0 +1,30 @@
+class Aflplusplus < Formula
+  desc "American fuzzy lop: Security-oriented fuzzer"
+  homepage "https://github.com/vanhauser-thc/AFLplusplus"
+  url "https://github.com/vanhauser-thc/AFLplusplus/archive/2.62c.tar.gz"
+  sha256 "cde181ac733aa3a1212ffcb494bb9306a2086c7521fb006719b0e15cd8015c63"
+
+  depends_on "automake" => :build
+  depends_on "wget" => :build
```

Comment:
Why does it need wget only on build time?

Reply:
there is a unicorn for AFL release to download during the build process.

Reply:
Make scripts shouldn't download anything when building. If you need a file you should use a resource block.

Reply:
Surely ... but the shell script checks the presence of wget anyway.

Reply:
That seems worth patching upstream.

Reply:
done it ... is cloning GitHub repo doable in a build ?

Reply:
Yes, but in this case you probably want to be applying a patch instead (using a `patch do` block with https://github.com/vanhauser-thc/AFLplusplus/commit/9ddd7e0e3f6f9dc8b837e96b508db7fd1f9a1a19.patch?full_index=1). When cloning a repo we would still want to follow the tag.

Reply:
Finally opted for building the essential bit, might need further upstream changes.